src/components/mainContent/ProgressBar.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "../../AppMain.css";
+import "../../index.css";
+import "../posts/posts.css";
+import "./mainContent.css";
+
+export default function ProgressBar(props) {
+
+    function clickHandler(n) {
+        props.setVisibleSections(n);
+        props.setCurrentVisibleText(0);
+        props.setForwardVisible(false);
+    }
+
+    const indicies = [0, 1, 3, 5, 7, 9]
+    let content = props.content
+        .map((section, index) => {
+            if (indicies.includes(index)) {
+                return (
+                    <button className='progress-button' onClick={() => clickHandler(index)}>
+                        <img className={`profile-pic no-margin ${index > props.visibleSections ? "gray" : ""}`} src={section.post.profilePic} alt={`shortcut: ${section.post.header}`}/>
+                    </button>
+                );
+            } else {
+                return ""; // added because of npm run build warning about a missing return
+            }
+        });
```

Comment:
Hm, this is a bit awkward - we aren't actually mapping items one-to-one with the `content` array, so `.map` may not be the right pattern. One of two things I would suggest:

1. use `.forEach`, though it's a bit clunkier
2. before you go between `content` and `map`, you can filter items out of the array ("map-filter-reduce") - see [`Array.prototype.filter()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter)